airflow/providers/postgres/operators/postgres.py

```diff
@@ -60,9 +60,9 @@ def __init__(
         self.autocommit = autocommit
         self.parameters = parameters
         self.database = database
-        self.hook = None
+        self.hook: Optional[PostgresHook] = None
 
-    def execute(self, context):
+    def execute(self, context: Dict[Any, Any]) -> None:
```

Comment:
Are you sure that `context` has a `Dict` type?

Reply:
Yes, good point man!
I dind't find exactly what is this variable, could it be better put just `Any`?

Reply:
Maybe yes